app/src/modules/collections/routes/collection.vue

```diff
@@ -127,7 +127,7 @@
 						<v-button secondary @click="confirmArchive = false">
 							{{ t('cancel') }}
 						</v-button>
-						<v-button class="action-archive" :loading="archiving" @click="archive">
+						<v-button class="action-archive warning" :loading="archiving" @click="archive">
```

Comment:
I'm wondering if we should align this with `v-notice` 🤔 

`v-notice` uses `type` prop for the same purpose. That wouldn't work for `v-button` though as type is already an existing attribute on `<button>` elements.

Reply:
Ref:



https://github.com/directus/directus/blob/0b17fdcf8163d82edd146e84c90bc2dafd6a9bde/app/src/components/v-notice/v-notice.vue#L13-L16

Reply:
I was thinking the same thing about the type property already being used. I could name it something like "style-type" on VButton.

Reply:
In lieu of `style` / `type`, lets use `kind` as the prop name, feels fairly natural: `<v-button kind="danger">` 

cc @benhaynes

Reply:
fixed in 34684b1